linux/.gitignore

```diff
@@ -0,0 +1,57 @@
+# linux
+autom4te.cache
+*.orig
+Makefile.in
+configure
+aclocal.m4
+config.guess
+config.sub
+config.h.in
+config.h.in~
+config.rpath
+compile
+depcomp
+install-sh
+ltmain.sh
+missing
+test-driver
+ylwrap
+INSTALL
+ABOUT-NLS
+ltoptions.m4
+ltsugar.m4
+ltversion.m4
+lt~obsolete.m4
+libtool.m4
+__pycache__
+*.egg-info
+build-*
+deb-*
+builddebs
+make_deb
+tmp/
+dist/
+*~
+linux/ibus-kmfl/m4/codeset.m4
```

Comment:
Should these now remove the linux/ prefix?

Reply:
good catch

Reply:
done